src/bundle/Resources/public/scss/functions/_div.scss

```diff
@@ -0,0 +1,3 @@
+@function div($number1, $number2) {
```

Comment:
I think it would be better to create a folder `polyfill` for this function here in admin-ui and import it in all needed bundles. Is it possible or there are some limitations in dart?